libp2p/NodeTable.cpp

```diff
@@ -514,16 +508,17 @@ void NodeTable::onReceived(UDPSocketFace*, bi::udp::endpoint const& _from, bytes
 			{
 				PingNode in = PingNode::fromBytesConstRef(_from, rlpBytes);
 				if (in.version != dev::p2p::c_protocolVersion)
+					return;
+				if (RLPXDatagramFace::secondsSinceEpoch() - in.ts > 3)
```

Comment:
Looking at the other places, you seem to have mixed in a change where the expiration stamp is now the time when the packet was sent.